src/repm-invoker.ts

```diff
@@ -23,7 +23,6 @@ export interface Invoke {
   <RPC extends keyof InvokeMap>(rpc: RPC, args: InvokeMap[RPC][0]): Promise<
     InvokeMap[RPC][1]
   >;
-  (rpc: string, args?: JSONValue): Promise<JSONValue>;
```

Comment:
I found this on my branch. This last case was allowing anything to be passed into invoke